_locales/_untranslated_en.json

```diff
@@ -23,5 +23,41 @@
   },
   "pref_current_account": {
     "message": "Current Account"
+  },
+  "onboarding_create_instant_account": {
+    "message": "Let's get started!"
```

Comment:
I am not sure "let's get started!" fully convey "I don't have account / register"

Reply:
What about "Create Account"? I guess people have already considered this wording

Reply:
yeah, this is a not-so-easy question.

the issue with "Create Account" (what would be "Create Profile" then) is that ppl need to understand or may ask theirselves what an "Account" or a "Profile" is, at the very first screen already, or if the can get along without? and this even if they never used Delta Chat before and the only thing they know is that someone said "Hey, we can use that for chatting". we're talking about ppl that do not know or care, that an email address will created

therefore, the very simple wording, speaking about nothing special, just "Let me in" - this is also what happens when you install WhatsApp (no verb), Signal (no verb, "Continue"), Telegram (no verb, "Start Messaging") and probably many others

so, if you will, we make things easier to use for ppl who do not know or care, maybe at the cost of ppl knowing that an account will be created. implicitly, however, more experienced ppl get by the "third button" that "sth" is created (which, then, thinking it over  should pick up the wording of the subsequent page, so maybe better "I Already Have a Account/Profile"  - so give the thing only one name, and skip this name where possible)

Reply:
let's leave wording/flow discussions outside this PR for now, even if things are not 100% perfect, we need to move forward